backtrace-library/publish.gradle

```diff
@@ -111,12 +111,10 @@ afterEvaluate { project ->
         }
 
         task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
-            classifier = 'javadoc'
```

Comment:
why are you removing that and below

Reply:
This is required by gradle upgrade to version 8.2.0